src/metrics/commits_per_pr/index.ts

```diff
@@ -33,11 +35,17 @@ export const collectCommitsPerPrMetrics = async (
     const data = request.data;
     numberOfCommits = data.length;
 
-    commit_timestamps = data.map(
-      ({ commit }) => moment.utc(commit.author?.date).valueOf() || []
-    );
+    // Checking the author date will not reflect amendments
+    commit_timestamps = data.map(({ commit }) => ({
+      authored: commit.author?.date
+        ? moment.utc(commit.author.date).valueOf()
+        : null,
+      committed: commit.committer?.date
+        ? moment.utc(commit.committer.date).valueOf()
+        : null,
+    }));
   } catch (error) {
-    commit_timestamps = [];
+    // TODO: send error output
```

Comment:
Did you maybe plan to use the logger here? If this is a TODO for a later task, you could remove the import of `logger` and `LogWarnings`again.